src/Chapter1.hs

```diff
@@ -301,31 +301,31 @@ expressions in GHCi
   functions and operators first. Remember this from the previous task? ;)
 
 >>> 1 + 2
-INSERT THE RESULT INSTEAD OF THE TEXT
+3              -- INSERT THE RESULT INSTEAD OF THE TEXT
```

Comment:
To pass doctests, you need to remove this extra text. The below should work now :slightly_smiling_face: 
```suggestion
3
```